istio/README.md

```diff
@@ -24,18 +24,14 @@ If you want to monitor the Envoy proxies in Istio, configure the [Envoy integrat
 
 ### Configuration
 
-Edit the `istio.d/conf.yaml` file (in the `conf.d/` folder at the root of your [Agent's configuration directory][7]) to connect to Istio. See the [sample istio.d/conf.yaml][8] for all available configuration options.
-
 #### Metric collection
-To monitor the `istiod` deployment and `istio-proxy` in Istio `v1.5+`, use the following configuration:
-    
-```yaml
-init_config:
+There are two key components involved in setting up the Istio integration to collect the Prometheus-formatted Istio metrics. Aligning with the [Istio architecture][23], there is the **data plane** (the `istio-proxy` sidecar containers) and the **control plane** (the `istiod` service managing the proxies). These are both run as `istio` Agent checks, but have different responsibilities, and they are  configured separately, as described below.
```

Comment:
Can we also continue mentioning this is for Istio `v1.5+`?

Reply:
Do we still need to make this distinction? Istio 1.4 and under received their end-of-life as of June 2020. I don't have numbers to back this, but customers should be on 1.5+ by now.  

That being said I did note this in the `Control plane configuration` section. Is that sufficient? If not I can add something like that here.

Reply:
I think it's worth it to keep it in considering 2020 was not too long ago.

Reply:
Added 1.5+ note back in